README.md

```diff
@@ -250,26 +298,27 @@ You can also use `--warmup`, `--bench`, or `--once` to set these environment var
 ./run_benchmarks.rb railsbench --once
 ```
 
-There is also a handy script for running benchmarks just once using
-`WARMUP_ITRS=0 MIN_BENCH_ITRS=1 MIN_BENCH_TIME=0`, for example
-with the `--yjit-stats` command-line option:
+You can also run a single benchmark directly with Ruby:
 
-```
-./run_once.sh --yjit-stats benchmarks/railsbench/benchmark.rb
+```bash
+# Run once with YJIT stats
+ruby --yjit-stats benchmarks/railsbench/benchmark.rb
```

Comment:
This doesn't run a single iteration so it's not "once", unlike the `./run_once.sh --yjit-stats benchmarks/railsbench/benchmark.rb` before